<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="5" hostname="LAPTOP-TUKU" failures="1" tests="1" name="Regression" time="13.259" errors="0" timestamp="2022-05-04T22:40:10 IST">
  <testcase classname="FuntionalTestCases.Sprint_1" name="Login" time="6.457">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at FuntionalTestCases.Sprint_1.Login(Sprint_1.java:39)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- Login -->
  <testcase classname="FuntionalTestCases.Sprint_1" name="TC03" time="0.0">
    <ignored/>
  </testcase> <!-- TC03 -->
  <testcase classname="FuntionalTestCases.Sprint_1" name="TC02" time="0.0">
    <ignored/>
  </testcase> <!-- TC02 -->
  <testcase classname="FuntionalTestCases.Sprint_1" name="TC06" time="0.0">
    <ignored/>
  </testcase> <!-- TC06 -->
  <testcase classname="FuntionalTestCases.Sprint_1" name="TC05" time="0.0">
    <ignored/>
  </testcase> <!-- TC05 -->
  <testcase classname="FuntionalTestCases.Sprint_1" name="TC04" time="0.0">
    <ignored/>
  </testcase> <!-- TC04 -->
</testsuite> <!-- Regression -->
